mne/stats/tests/test_parametric.py

```diff
@@ -112,3 +116,43 @@ def test_f_twoway_rm():
 
     fvals, _ = f_mway_rm(test_data, [8], 'A')
     assert_array_almost_equal(fvals, test_external['r_fvals_1way'], 5)
+
+
+@pytest.mark.parametrize('kind, kwargs', [
+    ('1samp', {}),
+    ('ind', {}),  # equal_var=True is the default
```

Comment:
Maybe I'm missing something, but for the new `ttest_ind_no_p` the default is `equal_var=False`, not `True` (unlike scipy). Isn't that a problem here?

Reply:
Yes that's a bug not caught by the tests (need the number of samples to vary to show it), fixed